clients/cloud/java/README.md

```diff
@@ -8,19 +8,21 @@ Produce messages to and consume messages from a Kafka cluster using the Java Pro
 * Java 1.8 or higher to run the demo application
 * Maven to compile the demo application
 
-To run this example, create a local file with configuration parameters to connect to your Kafka cluster, which can be on your local host, [Confluent Cloud](https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud), or any other cluster.
-If this is a Confluent Cloud cluster, you must have:
+To run this example, update the `example.config` file with configuration parameters to connect to your Kafka cluster, which can be on your local host, [Confluent Cloud](https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud), or any other cluster. If this is a Confluent Cloud cluster, you must have:
```

Comment:
It may be helpful to hyperlink all of example.config mentions.

For example, change:

```
`example.config`
```

to

```
[example.config](example.config)
```